python/hail/linalg/matrix.py

```diff
@@ -20,6 +23,42 @@ def read(cls, path):
         return cls(Env.hail().distributedmatrix.BlockMatrix.read(
             hc._jhc, path))
 
+    @staticmethod
+    @handle_py4j
+    def _from_numpy_matrix(numpy_matrix, block_size):
+        """Create a BlockMatrix from the given numpy matrix.
+
+        Notes
+        -----
+
+        This method is being used to create a BlockMatrix for testing BlockMatrix's entries_table() method.
+        It is not meant to be used for creating large matrices.
+
+        Parameters
+        ----------
+        numpy_matrix : numpy matrix
+            Numpy matrix to convert to a BlockMatrix.
+
+        Returns
+        -------
+        :class:`.BlockMatrix`
+            BlockMatrix.
+        """
+        # will be slow (used only for testing BlockMatrix explode method)
+        num_rows, num_cols = numpy_matrix.shape[0:2]
+        sc = Env.hc()._jsc
+        data = jarray(Env.jvm().double, np.ravel(numpy_matrix))
+        # np.ravel() exports row major, breeze is column major
```

Comment:
"exports row major by default"

(it actually can export in four different ways, but I think it's fine to use the default since that's how the data is stored)